g2p/mappings/langs/fin/fin_to_ipa.csv

```diff
@@ -0,0 +1,50 @@
+aa,ɑː,,
+ai,ɑj,,
+au,ɑʊ,,
+a,ɑ,,
+b,b,,
+c,s,,
+d,d,,
+ee,eː,,
+ei,ej,,
+eu,eʊ,,
+e,e,,
+f,f,,
+g,ɡ,,
+h,h,,
+ii,iː,,
+ie,je,,
+i,i,,
+j,j,,
+k,k,,
+l,l,,
+m,m,,
+n,n,,
+oo,oː,,
+ou,oʊ,,
```

Comment:
wow - surprisingly straightforward! I won't look into this, I'll just take your word it's this transparent!

Reply:
A native speaker could certainly clarify some of the vowels a bit, and there are some nuances of gemination / lengthening in some contexts, but yes, it's basically IPA :)

Note that this is the literary language, what people actually speak is ... different